codegen/consts.js

```diff
@@ -2,42 +2,66 @@ module.exports.props = require("react-html-attributes");
 module.exports.voids = ["area", "base", "br", "col", "embed", "hr", "img", "input", "link", "meta", "param", "source", "track", "wbr"];
 module.exports.types = {
   "allowFullScreen": "Boolean",
+  "allowTransparency": "Boolean",
   "async": "Boolean",
+  "autoComplete": "Boolean",
+  "autoFocus": "Boolean",
   "autoPlay": "Boolean",
   "capture": "Boolean",
+  "cellPadding": "Number",
+  "cellSpacing": "Number",
   "checked": "Boolean",
   "children": "Array JSX",
   "cols": "Number",
+  "colSpan": "Number",
+  "contentEditable": "Boolean",
   "controls": "Boolean",
+  "dangerouslySetInnerHTML": "{ __html :: String }",
   "default": "Boolean",
   "defer": "Boolean",
   "disabled": "Boolean",
+  "draggable": "Boolean",
   "formNoValidate": "Boolean",
+  "frameBorder": "Number",
   "hidden": "Boolean",
   "itemScope": "Boolean",
   "loop": "Boolean",
+  "marginHeight": "Number",
+  "marginWidth": "Number",
+  "maxLength": "Number",
   "max": "Number",
+  "minLength": "Number",
   "min": "Number",
   "multiple": "Boolean",
   "muted": "Boolean",
+  "noValidate": "Boolean",
+  "onBlur": "EventHandler",
+  "onChange": "EventHandler",
   "onClick": "EventHandler",
+  "onFocus": "EventHandler",
   "onInput": "EventHandler",
   "onInvalid": "EventHandler",
+  "onKeyDown": "EventHandler",
+  "onKeyPress": "EventHandler",
+  "onKeyUp": "EventHandler",
   "onSubmit": "EventHandler",
-  "noValidate": "Boolean",
-  "onChange": "EventHandler",
   "open": "Boolean",
   "playsInline": "Boolean",
   "readOnly": "Boolean",
   "required": "Boolean",
   "reversed": "Boolean",
-  "rowSpan": "Number",
   "rows": "Number",
+  "rowSpan": "Number",
   "scoped": "Boolean",
   "seamless": "Boolean",
   "selected": "Boolean",
   "size": "Number",
   "span": "Number",
-  "start": "Number"
+  "spellCheck": "Boolean",
+  "srcDoc": "JSX",
+  "start": "Number",
+  "suppressContentEditableWarning": "Boolean",
+  "tabIndex": "Number",
```

Comment:
`Int` as well

Reply:
Awaiting a decision on the general Int vs Number thing!